x/bank/README.md

```diff
@@ -1032,3 +1036,13 @@ Example Output:
   }
 }
 ```
+
+## concepts/changes specific to heimdall-v2
+
+Since heimdall is a unique component of the PoS architecture and not a traditional Cosmos SDK app, some changes had to be made in order to ensure correct functionality, some of which are :
+
+* MsgSetSendEnabled is not supported since matic is the only denom used. 
+
+* (Un)delegation is not supported since staking related logic is handled by PoS [core contracts](https://github.com/0xPolygon/core-contracts/tree/main/contracts) deployed on Ethereum.
```

Comment:
Unbonding (undelegation) is a staking tx, so probably best this information lives in a higher level spec/readme or in modifications to `x/staking` readme.

Reply:
Yeah I understand. That's why I kept it under the `concepts/changes specific to heimdall-v2` sub-heading to highlight that this is specific to heimdall. Since bank keeper contains methods for (un)delegating coins I decided to mention it in the README as well.

Reply:
Fair enough, I don't think you need to worry about conforming to the BaseKeeper interface, since the bank keeper method for crediting accounts in an unbonding event isn't a public method. I would make sure to track somewhere any place this gets called internally.